### ISSUE TITLE
Fixing static declaration error

### ISSUE THREAD

**Issue by AKuHAK** [ps2dev/ps2sdk#95]
Modern toolchain produces this error - it seems that both declarations should be static or vice-versa - both non-static.
```cpp
make -C tcpips
make[3]: Entering directory '/work/ee/rpc/tcpips'
mkdir -p obj/
mips64r5900el-ps2-elf-gcc -D_EE -G0 -O2 -Wall  -Isrc/ -Isrc/include -Iinclude/ -I/work/ee/kernel/include -I/work/common/include -I/work/ee/libc/include -I/work/ee/erl/include -DPS2IP_DNS -c src/ps2ipc.c -o obj/ps2ipc.o
src/ps2ipc.c:58:24: error: static declaration of 'ip_addr_any' follows non-static declaration
 static const ip_addr_t ip_addr_any = IPADDR4_INIT(IPADDR_ANY);
                        ^~~~~~~~~~~
In file included from include/ps2ips.h:19,
                 from src/ps2ipc.c:21:
/work/common/include/tcpip.h:834:24: note: previous declaration of 'ip_addr_any' was here
 extern const ip_addr_t ip_addr_any;
                        ^~~~~~~~~~~
```

**Comment by AKuHAK** [comment]
@sp193 can you comment on this issue? I can provide patch but im not actually sure if static declaration should be removed in both cases or should be kept in both cases?

**Comment by sp193** [comment]
This has been fixed in d394702.

**Comment by sp193** [comment]
So to answer the question: the symbol within ps2ipc.c should not be static, otherwise the macro(s) within tcpip.h would not work.
